config.xhgui.yaml

```diff
@@ -0,0 +1,11 @@
+#ddev-generated
+hooks:
+  post-start:
+  # Create a new database called "xhgui"
```

Comment:
I guess the title of this PR should be "Use default DDEV database server" then, right?